src/mock-graphql/typeDefs.ts

```diff
@@ -13,13 +13,14 @@ export const typeDefs = gql`
 
     type Submission {
         id: ID!
-        title: String!
         updated: String!
         articleType: String!
-        author: AuthorDetails
-        coverLetter: String
+        author: AuthorDetails!
         manuscriptFile: File
         manuscriptDetails: ManuscriptDetails
+        supportingFiles: [File]
+        coverLetter: String
+        suggestions: [String]!
```

Comment:
Can we have suggestions as optional?

Reply:
Done

Reply:
I think it should just be an empty array if there are no suggestions. so current definition seems correct